src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/SourceInfo.cs

```diff
@@ -45,12 +46,18 @@ public SourceInfo(string fullTypeName, bool isInterface, ImmutableHashSet<string
         /// </summary>
         public ImmutableHashSet<string> TaintedMethods { get; }
 
+        /// <summary>
+        /// Indicates array of this type generates tainted data.
```

Comment:
>Indicates array of this type generates tainted data. [](start = 12, length = 52)

Only if the array is initialized with constant values, right?

"Indicates arrays initialized with constant values of this type generates tainted data."